library/SimplePie/Item.php

```diff
@@ -90,7 +90,26 @@ class SimplePie_Item
 	public function __construct($feed, $data)
 	{
 		$this->feed = $feed;
-		$this->data = $data;
+		$this->data = $this->array_map_recursive('trim', $data);
```

Comment:
**Where does it make sense to return an untrimmed whitespace?**
Sorry if I don't see it, but I don't think an trim would have any negative effect on any of the affected functions. On the positive side, we won't need to trim anything multiple times, just because we're using the data in multiple places... #performance

Here's the full list:
`get_authors`, `get_base`, `get_categories`, `get_content`, `get_contributors`, `get_copyright`, `get_date`, `get_description`, `get_enclosures`, `get_id`, `get_item_tags`, `get_latitude`, `get_links`, `get_longitude`, `get_source`, `get_thumbnail`, `get_title`, `get_updated_date`.